W-16110978 - update docs with clarity for port-name mapping (#2632)
mulesoft/docs-connectors: sap/5.9/modules/ROOT/pages/sap-connector-reference.adoc

| Name | Type | Description | Default Value | Required
| Configuration | String | Name of the configuration to use. | | x
| Gateway Host a| String |  Name of the host running the gateway server. This is the name of the host running the gateway handler. SAP Gateway is a development framework used to connect non-SAP applications to SAP applications. You can configure the SAP Gateway host to be the trusted system using an RFC destination.|  | x

> | Gateway Service a| String |  Service port for the gateway service of the SAP system. To use name instead of port, additional mapping of gateway service name :: gateway service port is needed.  |  | x

Reviewer: ```suggestion
| Gateway Service a| String |  Service port for the gateway service of the SAP system. To use the name instead of the port, you must map the gateway service name to the gateway service port.  |  | x
```

| Name | Type | Description | Default Value | Required
| Configuration | String | Name of the configuration to use. | | x
| Gateway Host a| String |  Name of the host running the gateway server. This is the name of the host running the gateway handler. SAP Gateway is a development framework used to connect non-SAP applications to SAP applications. You can configure the SAP Gateway host to be the trusted system using an RFC destination.|  | x

> | Gateway Service a| String |  Service port for the gateway service of the SAP system. To use name instead of port, additional mapping of gateway service name :: gateway service port is needed. |  | x

Reviewer: ```suggestion
| Gateway Service a| String |  Service port for the gateway service of the SAP system. To use the name instead of the port, you must map the gateway service name to the gateway service port.  |  | x

```
